src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -355,6 +355,24 @@
             az acr build-task list -r MyRegistry -o table
 """
 
+helps['acr build-task list-arguments'] = """
```

Comment:
@djyou is this a common az cli pattern, instead of just returning normal json and --query for properties instead? I feel like it clutters the CLI significantly by first classing these commands

Reply:
This is a different API that shows sensitive information (e.g., secrets in build args), which wasn't included in get build task path.

We can combine this with `acr build-task get` though, with a flag.